katharsis-jpa/src/main/java/io/katharsis/jpa/query/JpaFilterOperators.java

```diff
@@ -0,0 +1,59 @@
+package io.katharsis.jpa.query;
+
+import io.katharsis.queryspec.FilterOperator;
+import io.katharsis.queryspec.FilterOperatorRegistry;
+
+public class JpaFilterOperators {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)